content/documentation/shortcodes.md

```diff
@@ -0,0 +1,213 @@
+---
+author: dpordomingo
+date: 2017-10-31
+title: "Enforcing a maintainable layout using shortcodes"
+image: /post/shortcodes/header.png
+description: "Use reusable shortcodes instead of custom HTML code to ensure a maintainable layout"
+categories: ["documentation"]
+---
+
+Since we're trying to have a maintainable blog, with a robust layout that should be easily redesigned, we decided to forbid HTML code inside the blog posts.
```

Comment:
s/maintainable blog, with/maintainable blog with/
s/should be easily/can be easily/